lib/chef/knife/google_disk_create.rb

```diff
@@ -19,10 +19,12 @@
 
 require "chef/knife"
 require "chef/knife/cloud/command"
+require_relative "cloud/google_service_helpers"
```

Comment:
Can we try moving both of these into the deps block in each plugin so we're lazy loading it?